src/models/syncRule.ts

```diff
@@ -203,6 +358,7 @@ export default class SyncRule {
                       }
                     : null,
             ]),
+            includeExclude: [],
```

Comment:
[minor] Why don't we use metadataIncludeExclude for the validation key? Here we only have Metadata but when porting to data-Sync could be confusing.

Reply:
Done!